datafusion/physical-expr/src/functions.rs

```diff
@@ -57,7 +58,7 @@ pub fn create_physical_expr(
     fun: &BuiltinScalarFunction,
     input_phy_exprs: &[Arc<dyn PhysicalExpr>],
     input_schema: &Schema,
-    execution_props: &ExecutionProps,
+    _execution_props: &ExecutionProps,
```

Comment:
Here the `ExecutionProps` param can be moved, but I'd like to do it in a follow-up PR because doing all in a PR may make me less easy to inspect error.

Reply:
Yes, it doesn't need this param now, but before it needs this for BuiltInFunction to create physical function for some Date function like to_date. Since these functions has been ported, it will not need this param anymore.